pythclient/utils.py

```diff
@@ -0,0 +1,33 @@
+import ast
+import dns.resolver
+from loguru import logger
+from typing import Optional
+
+DEFAULT_VERSION = "v2"
+
+
+def get_key(network: str, type: str, version: str = DEFAULT_VERSION) -> Optional[str]:
```

Comment:
Please add a comment here for future use:
Retrieving keys via DNS TXT records should not be considered secure and is provided as a convenience only. Accounts should be stored locally and verified before being used for production.